test/literate001/expected

```diff
@@ -1,4 +1,9 @@
-./test003a.lidr:1:0:Program line next to comment
+./test003a.lidr:1:0:
```

Comment:
Why is `./` not stripped here, but is stripped somewhere else?

Reply:
That is a good question!  o_O . I'll make a note to look later.